portal-api/src/constants/date-formats.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  DD: 'dd',
+  MM: 'MM',
+  YYYY: 'yyyy',
+};
```

Comment:
I think this will be confusing because capitalised `DD` and `YYYY` mean something different to their lower-case equivalents. Could we include have constants for common formats? e.g. `dd MMM yyyy` and `dd MMMM yyyy`?